shared/constants/stablecoins.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Used for determining optimal slippage in swaps
+ * Only includes USDC and USDT for now
+ */
+
+import { CHAIN_IDS } from './network';
+
+/**
+ * A map of chain IDs to sets of known stablecoin contract addresses with deep liquidity.
+ * Used to determine if a pair qualifies for lower default slippage to avoid frontrunning.
+ * Just using USDC and USDT for now, but can add more as needed.
+ */
+export const StablecoinsByChainId: Partial<Record<string, Set<string>>> = {
+  [CHAIN_IDS.MAINNET]: new Set([
```

Comment:
Nit: use constant case

```suggestion
export const STABLECOINS_BY_CHAIN_ID: Partial<Record<string, Set<string>>> = {
  [CHAIN_IDS.MAINNET]: new Set([
```